### DISCUSSION TITLE
editor.setComponents(html) not load JS scripts  each time

### DISCUSSION THREAD

**Discussion by NoumanAhmad-Braqsol** [GrapesJS/grapesjs#3701] [General]
hi @artf , Thanks in advance . I am using  
` editor.setComponents(content);`
Here i am passing the` content` having html with external CSS and external JS script tag
When i load the page first time and use editor.setComponents(content) it works fine and load all CSS and JS scripts .

Now if i call it again and pass the same content this time only html and CSS work . it does not load any external JS script.
if i refresh the page and call again it will again work fine . Can you please explain this behavior and the solution.

**Comment by artf** [comment]
Can you create a reproducible demo?
